torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -13,9 +13,20 @@
 namespace torch {
 namespace jit {
 
-static bool texpr_fuser_enabled = true;
+static bool texpr_fuser_enabled_ = false;
 void setTensorExprFuserEnabled(bool val) {
-  texpr_fuser_enabled = val;
+  texpr_fuser_enabled_ = val;
+}
+
+static bool tensorExprFuserEnabled() {
+  static const char* enable_c_str = std::getenv("PYTORCH_TENSOREXPR");
+  if (!enable_c_str) {
+    return texpr_fuser_enabled_;
+  }
+  if (std::string(enable_c_str) == "0") {
+    return false;
+  }
+  return true;
```

Comment:
If `PYTORCH_TENSOREXPR` env variable is set then `torch._C._jit_set_texpr_fuser_enabled(False/True)` calls will have no effect. Is it a desired behavior?

Reply:
for me, yea. One can override the program with `PYTORCH_TENSOREXPR=0`

Right now, for benchmarks, this is really useful because I don't need to inspect internals of various scripts.  In the future, I see the python interface as eventually becoming deprecated anyway (once it is always on), and this functioning the same way regardless